src/hud.cpp

```diff
@@ -87,24 +87,31 @@ Hud::Hud(video::IVideoDriver *driver, scene::ISceneManager* smgr,
 	m_halo_boxes.clear();
 
 	m_selection_pos = v3f(0.0, 0.0, 0.0);
-	std::string mode = g_settings->get("node_highlighting");
+	std::string mode_setting = g_settings->get("node_highlighting");
```

Comment:
Look, IMO it's not necessary here because it's in the constructor and will not affect performance at all. But if it makes people happy just change it ;)